shared/teams/team/index.tsx

```diff
@@ -5,82 +5,175 @@ import * as Types from '../../constants/types/teams'
-      default: {
-        throw new Error(`Impossible case encountered in team page list: ${item}`)
-      }
-    }
+      ) : (
+        <TeamTabs teamID={teamID} selectedTab={selectedTab} setSelectedTab={setSelectedTab} />
+      ),
   }
 
-  const renderSectionHeader = ({section}) => (section.header ? renderItem({item: section.header}) : null)
+  const sections: Array<Section> = [headerSection]
+  const membersSections = useMembersSections(teamID, teamMeta, teamDetails, yourOperations)
+  const botSections = useBotSections(teamID, teamMeta, teamDetails, yourOperations)
+  const invitesSections = useInvitesSections(teamID, teamDetails)
+  const channelsSections = useChannelsSections(teamID, selectedTab === 'channels')
+  const subteamsSections = useSubteamsSections(teamID, teamDetails, yourOperations)
 
-  const SectionList = Styles.isMobile ? Kb.ReAnimated.createAnimatedComponent(Kb.SectionList) : Kb.SectionList
+  switch (selectedTab) {
+    case 'members':
+      if (yourOperations.manageMembers && flags.teamsRedesign) {
+        sections.push(...invitesSections)
+      }
+      sections.push(...membersSections)
+      break
+    case 'bots':
+      sections.push(...botSections)
+      break
+    case 'invites':
+      sections.push(...invitesSections)
+      break
+    case 'settings':
+      sections.push({data: ['settings'], key: 'settings', renderItem: () => <Settings teamID={teamID} />})
+      break
+    case 'channels':
+      sections.push(...channelsSections)
+      break
+    case 'subteams':
+      sections.push(...subteamsSections)
+      break
+  }
+
+  // Animation
+  const SectionList: typeof Kb.SectionList = Styles.isMobile
+    ? Kb.ReAnimated.createAnimatedComponent(Kb.SectionList)
+    : Kb.SectionList
   const offset = Styles.isMobile ? new Kb.ReAnimated.Value(0) : undefined
   const onScroll = Styles.isMobile
     ? Kb.ReAnimated.event([{nativeEvent: {contentOffset: {y: offset}}}], {useNativeDriver: true})
     : undefined
-  const selectionPopupTabName =
-    props.selectedTab === 'members' ? 'teamMembers' : props.selectedTab === 'channels' ? 'teamChannels' : ''
 
+  const renderSectionHeader = ({section}) =>
+    section.title ? (
+      <Kb.SectionDivider
+        label={section.title}
+        collapsed={section.collapsed}
+        onToggleCollapsed={section.onToggleCollapsed}
+      />
+    ) : null
   return (
     <Kb.Box style={styles.container}>
-      <TeamsSubscriber />
-      <TeamDetailsSubscriber teamID={props.teamID} />
-      {Styles.isMobile && flags.teamsRedesign && <MobileHeader teamID={props.teamID} offset={offset} />}
+      {Styles.isMobile && flags.teamsRedesign && <MobileHeader teamID={teamID} offset={offset} />}
       <SectionList
-        renderItem={renderItem}
         renderSectionHeader={renderSectionHeader}
         stickySectionHeadersEnabled={Styles.isMobile}
-        sections={props.sections}
-        style={styles.list}
+        sections={sections}
```

Comment:
The style previously pushed the list down so the header was visible, it covers the header now.